src/main.rs

```diff
@@ -25,6 +25,7 @@ struct Args {
 
 #[actix_rt::main]
 async fn main() -> Result<(), Box<dyn Error>> {
+    println!("Go...");
```

Comment:
I think this is good to have here at the very beginning, but let's make it "syncstorage-rs is starting up"

Reply:
actually, it's crap. I had added it to see where the seg fault was happening.